### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/de/app.json [hunk 1]
```diff
@@ -483,23 +480,21 @@
         },
         "privacyPolicy": "MoonPay-Datenschutzrichtlinie",
         "protection": {
-          "description": "Die BitBoxApp sammelt beim Kauf von {{name}} keine Daten. Die eingehenden Transaktionen werden wie reguläre Transaktion behandelt. MoonPay muss einige persönliche Daten sammeln, um funktionieren zu können. In deren Datenschutzbestimmungen wird ausführlich erläutert, wie mit diesen Daten umgegangen wird.",
-          "descriptionGeneric": "Die BitBoxApp erhebt beim Kauf von {{name}} keinerlei Daten, die eingehenden Gelder werden wie eine normale Transaktion behandelt. Partnerbörsen müssen jedoch einige Informationen sammeln, um zu funktionieren. Einzelheiten zum Umgang mit Daten kannst du der jeweiligen Datenschutzerklärung entnehmen.",
+          "descriptionGeneric_bitcoin": "Die BitBoxApp erhebt beim Kauf von Bitcoin keinerlei Daten, die eingehenden Gelder werden wie eine normale Transaktion behandelt. Partnerbörsen müssen jedoch einige Informationen sammeln, um zu funktionieren. Einzelheiten zum Umgang mit Daten kannst du der jeweiligen Datenschutzerklärung entnehmen.",
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:483]: hmmmmmm.... I think I accidentally nuked this and pasted an older verion. cc @sutterseba do you know which version is newer?

**Comment by thisconnect** [comment]
@benma LGTM, but we pinged translators to take a look at SL and CS lang then you need to pull again..

You could merge this however so Ci on master passes again.

**Comment by benma** [comment]
> @benma LGTM, but we pinged translators to take a look at SL and CS lang then you need to pull again..
> 
> You could merge this however so Ci on master passes again.

You need to approve first :)

**Review by thisconnect — APPROVED** [review]
LGTM 😇

**Comment by thisconnect** [comment]
@benma please pull again

**Comment by benma** [comment]
@thisconnect pulled

File: frontends/web/src/locales/cs/app.json [hunk 2]
```diff
@@ -483,23 +480,23 @@
         },
         "privacyPolicy": "Zásady ochrany osobních údajů MoonPay",
         "protection": {
-          "description": "BitBoxApp při nákupu {{name}} neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. MoonPay potřebuje ke své činnosti shromažďovat některé osobní údaje. Jejich Zásady ochrany osobních údajů podrobně vysvětlují, jak je s těmito údaji nakládáno.",
-          "descriptionGeneric": "BitBoxApp při nákupu {{name}} neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. Partnerské burzy však ke svému fungování potřebují shromažďovat určité informace. Podrobnější informace o tom, jak je s daty nakládáno, naleznete v jejich zásadách ochrany osobních údajů.",
+          "descriptionGeneric_bitcoin": "BitBoxApp při nákupu kryptoměn neshromažďuje žádné údaje, s příchozími prostředky se zachází jako s běžnou transakcí. Partnerské burzy však ke svému fungování potřebují shromažďovat určité informace. Podrobnější informace o tom, jak je s těmito údaji nakládáno, naleznete v jejich zásadách ochrany osobních údajů.",
```
Comment (thisconnect) [inline frontends/web/src/locales/cs/app.json:483]: @benma sorry there is still crypto in the text. this should be fixed in locize, if you have time ideally please pull one more time... 
if it is too late I think it is fine as is for this release.

**Pull request merged**